README.md

```diff
@@ -1,44 +1,38 @@
-# curriculum-databases-projects-template
+# Vet clinic Database
 
-> This template should be used for database related projects at Microverse.
-> Generate your own repository, update this README and edit all files content while working on projects. You should not be adding any new files unless asked otherwise.
+> This is a database for a vet clinic. It contains information about animals and their owners.
+> It also contains information about the vets and the treatments they have done.
 
 
 ## Getting Started
 
 This repository includes files with plain SQL that can be used to recreate a database:
 
-- Use [schema.sql](./schema.sql) to create all tables.
-- Use [data.sql](./data.sql) to populate tables with sample data.
-- Check [queries.sql](./queries.sql) for examples of queries that can be run on a newly created database. **Important note: this file might include queries that make changes in the database (e.g., remove records). Use them responsibly!**
-
+- `schema.sql`: contains the database schema in SQL.
+- `data.sql`: contains some data to insert into the database.
+- `queries.sql`: contains some queries that can be run against the database.
 
 ## Authors
 
-👤 **Author1**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
-
-👤 **Author2**
+👤 **Benjamin Kisenge (Me)**
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+* GitHub: [@benjamin kisenge](https://github.com/iambenkis)
+* Twitter: [@benjamin kisenge](https://twitter.com/iambenkis)
+* LinkedIn: [benjamin kisenge](https://www.linkedin.com/in/ben-kisenge/)
 
 ## 🤝 Contributing
 
 Contributions, issues, and feature requests are welcome!
 
-Feel free to check the [issues page](../../issues/).
+Feel free to check the [issues page](https://github.com/iambenkis/Vet-clinic-database/issues).
 
 ## Show your support
 
 Give a ⭐️ if you like this project!
 
 ## Acknowledgments
 
+- Microverse
 - Hat tip to anyone whose code was used
 - Inspiration
 - etc
```

Comment:
- [ ] You have added a link to your MIT.md file which is your license file. 👍 But i guess you forgot to add an MIT.md file in your repository 😄 No worries you can add this file using [microverse MIT.md file template](https://github.com/microverseinc/readme-template/blob/master/MIT.md)
- [ ] After adding MIT.md file in your repository it would be nice if you would add your **name**, **current year** and **app-type** because this is your readme file so this project should be licensed to you 🤔